Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1

```diff
@@ -9,7 +9,8 @@
     Use HTTPS for App Services.
 
 # Prerequisites:
-    Contributor and higher privileges on the App Services in a Subscription.
+    1. Contributor and higher privileges on the App Services in a Subscription.
+    2. AzAccount must be connected.
```

Comment:
Must be connected to Azure with an authenticated account

Reply:
Updated.